### PR TITLE
The Overflow, Nomai, Taco Tuesday files. Add icons.

### PR DESCRIPTION
**Pull request by Rdzanuwu**

### CODE REVIEW

File: HTML/Nomai.html [hunk 1]
```diff
@@ -0,0 +1,133 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="utf-8">
```
Comment (MrPeanut1028) [inline HTML/Nomai.html:5]: This line can be removed.

File: HTML/Nomai.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                    <li>On one specific planet, travelling to another specific planet will take you to the sixth location.</li>
+                    <li>Some interactions will change the color of the timer, or will incur a strike.</li>
+                    <li>To solve the module, you must end the twenty-two second timer at the sixth location with a specific timer color.</li>
+                    <li>Travelling away from the sixth location will incur a strike.</li>
+                    <li>You will never be unable to find required information before you deactivate the time looping mechanism. If you can't, assume nothing happens.</li>
+                </ul>
+            </div>
+            <div class="page-footer relative-footer">Page 2 of 3</div>
+        </div>
+                <div class="page page-bg-03">
```
Comment (MrPeanut1028) [inline HTML/Nomai.html:105]: This spacing needs to be adjusted / fixed.

File: HTML/Taco Tuesday.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        </div>
+        <div class="page page-bg-03">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Taco Tuesday</span>
+            </div>
+            <div class="page-content">
+                <ul><li>Otherwise, the final field is Friday.</li>
+                </ul>
+                <p>However, if both Day Fields are the same day, press the button between them to turn the final field off. We do not like redundant redundancies.</p>
```
Comment (MrPeanut1028) [inline HTML/Taco Tuesday.html:89]: This spacing / lining up of these lines could be fixed, although it isn't 100% necessary.

File: HTML/The Overflow.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+						</tr>
+					</table>
+					
+					<ul>
+						<li>If the number you get exceeds 10, subtract 10 until it isn't. If it is below 0, add 10 until it isn't.</li>
+                        <li>Press the button again when the last digit of the seconds timer is the newly obtained number.</li>
+                    </ul>
+
+                    
+                </div>
```
Comment (MrPeanut1028) [inline HTML/The Overflow.html:66]: Spacing & lining up is messy throughout this file, it needs to be cleaned up.

File: JSON/Taco Tuesday.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "Author": "Raddicus",
+  "Compatibility": "Untested",
+  "DefuserDifficulty": "Easy",
+  "Description": "Set a reminder for an event by properly inputting the name of the event (Food and Day) and the day it will happen. three-displays,",
```
Comment (MrPeanut1028) [inline JSON/Taco Tuesday.json:5]: Tags should be fixed here, that's incorrect (especially the comma at the end, I don't see what that's for).

File: JSON/The Overflow.json [hunk 6]
```diff
@@ -0,0 +1,24 @@
+{
+  "Author": "Deaf, kavinkul",
+  "Compatibility": "Compatible",
+  "Contributors": {
+    "Developer": [ "Deaf" ],
+    "Maintainer": [ "Deaf" ],
+    "Manual": [ "Deaf" ]
+  },
```
Comment (MrPeanut1028) [inline JSON/The Overflow.json:8]: The "Contributors" section needs to be removed since the "Author" section is being used.

**Review by MrPeanut1028 — CHANGES_REQUESTED** [review]

File: JSON/Taco Tuesday.json (hunk 5 above)
Comment (Rdzanuwu) [reply JSON/Taco Tuesday.json:5]: I dont understand whats wrong here.

Comment (Rdzanuwu) [reply JSON/Taco Tuesday.json:5]: You want me to delete  three-displays,?

Comment (MrPeanut1028) [reply JSON/Taco Tuesday.json:5]: You could do that, or you could make it like tags are supposed to be (`Tags: three-displays, buttons, numbers`).

Comment (Rdzanuwu) [reply JSON/Taco Tuesday.json:5]: I got a bit confused because i opend random other json and there wasnt a Tags: think. Ok then

**Review by MrPeanut1028 — APPROVED** [review]

**Pull request merged**